[2024-12-15T20:43:20.405-0800] {processor.py:186} INFO - Started process (PID=44465) to work on /Users/rishukhurana/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-15T20:43:20.407-0800] {processor.py:914} INFO - Processing file /Users/rishukhurana/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-15T20:43:20.408-0800] {logging_mixin.py:190} INFO - [2024-12-15T20:43:20.408-0800] {dagbag.py:588} INFO - Filling up the DagBag from /Users/rishukhurana/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-15T20:43:20.436-0800] {logging_mixin.py:190} INFO - [2024-12-15T20:43:20.435-0800] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/rishukhurana/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-15T20:43:20.436-0800] {logging_mixin.py:190} INFO - [2024-12-15T20:43:20.436-0800] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-15T20:43:20.437-0800] {processor.py:927} WARNING - No viable dags retrieved from /Users/rishukhurana/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-15T20:43:20.509-0800] {processor.py:208} INFO - Processing /Users/rishukhurana/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.108 seconds
[2024-12-16T08:55:34.628-0800] {processor.py:186} INFO - Started process (PID=44792) to work on /Users/rishukhurana/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-16T08:55:34.630-0800] {processor.py:914} INFO - Processing file /Users/rishukhurana/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-16T08:55:34.631-0800] {logging_mixin.py:190} INFO - [2024-12-16T08:55:34.631-0800] {dagbag.py:588} INFO - Filling up the DagBag from /Users/rishukhurana/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-16T08:55:34.671-0800] {logging_mixin.py:190} INFO - [2024-12-16T08:55:34.670-0800] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/rishukhurana/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-16T08:55:34.671-0800] {logging_mixin.py:190} INFO - [2024-12-16T08:55:34.671-0800] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-16T08:55:34.671-0800] {processor.py:927} WARNING - No viable dags retrieved from /Users/rishukhurana/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-16T08:55:34.751-0800] {processor.py:208} INFO - Processing /Users/rishukhurana/anaconda3/lib/python3.11/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.128 seconds
